src/.vuepress/components/SDKSelector.vue

```diff
@@ -93,13 +93,17 @@ export default {
     generateLink(item) {
       let method = '';
       let path = '';
+      const oldSDK = ['js5', 'php3', 'java2', 'android3'];
```

Comment:
I would prefer some kind of flag in the sdk.json file instead of hardcoded values